demo/index.html

```diff
@@ -1,133 +1,133 @@
 <!doctype html>
-<html>
-<head>
-	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
 
-	<title>Cosmoz Behaviors</title>
+		<title>cosmoz-tabs demo</title>
 
-	<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-	<link rel="import" href="../../polymer/polymer.html">
-	<link rel="import" href="../cosmoz-behaviors.html">
-</head>
-<body>
-	<template is="dom-bind" id="t">
-		<cosmoz-templatehelper-test></cosmoz-templatehelper-test>
-		<cosmoz-common-test></cosmoz-common-test>
-		<cosmoz-moneyhelper-test></cosmoz-moneyhelper-test>
-	</template>
-	<dom-module id="cosmoz-templatehelper-test">
-		<template>
-			<h3>Template helper test</h3>
-			<div>Empty string isEmpty: {{ isEmpty("") }}</div>
-			<div>Empty array isEmpty: {{ isEmpty(emptyArray) }}</div>
-			<div>concat 'hel', 'lo': {{ concat('hel', 'lo') }}</div>
-		</template>
-		<script type="text/javascript">
-			/*global Polymer, Cosmoz*/
-			'use strict';
-			Polymer({
-				behaviors: [
-					Cosmoz.TemplateHelperBehavior,
-				],
-				is: 'cosmoz-templatehelper-test',
-				properties: {
-					emptyArray: {
-						type: Array,
-						value: function () {
-							return [];
-						}
-					}
-				}
-			});
-		</script>
-	</dom-module>
+		<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
 
-	<dom-module id="cosmoz-common-test">
-		<template>
-			<h3>Common test</h3>
-			<div>Empty string isEmpty: {{ isEmpty("") }}</div>
-			<div>Empty array isEmpty: {{ isEmpty(emptyArray) }}</div>
-			<div>concat 'hel', 'lo': {{ concat('hel', 'lo') }}</div>
-		</template>
-		<script type="text/javascript">
-			/*global Polymer, Cosmoz*/
-			'use strict';
-			Polymer({
-				behaviors: [
-					Cosmoz.CommonBehaviors,
-				],
-				is: 'cosmoz-common-test',
-				properties: {
-					emptyArray: {
-						type: Array,
-						value: function () {
-							return [];
-						}
-					}
-				}
-			});
-		</script>
-	</dom-module>
+		<link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html" />
+		<link rel="import" href="../../iron-demo-helpers/demo-snippet.html" />
+		<link rel="import" href="../../iron-flex-layout/iron-flex-layout-classes.html" />
+
+		<link rel="import" href="./helpers/demo-templatehelper.html" />
+		<link rel="import" href="./helpers/demo-moneyhelper.html" />
+
+		<custom-style>
+			<style is="custom-style" include="demo-pages-shared-styles iron-flex">
+				#container {
+					 max-width: 800px;
+				 }
+			</style>
+		</custom-style>
+	</head>
+	<body unresolved>
+		<div id="container" class="vertical-section-container centered">
+			<h3><code>Cosmoz.TemplateHelperBehavior</code> Demo</h3>
+			<demo-snippet>
+				<demo-templatehelper></demo-templatehelper>
+				<template is>
```

Comment:
I guess this is a hack to render some content in the demo snippet.
Is this documented anywhere ?
Could you just add a comment above to explain why you're using `<template is>`, otherwise someone else could be tempted by deleting the `is`.

Reply:
I guess this is kind of a hack but `demo-snippet` only renders the template if it doesn't have the `is` attribute as you can see at
 https://github.com/PolymerElements/iron-demo-helpers/blob/master/demo-snippet.html .
`demo-snippet` doesn't yet have support to render something but output a different code and in examples related to behaviors this need to create a element using the behavior and showing how that element is created.